content/docs/web-components.md

```diff
@@ -20,14 +20,14 @@ class HelloMessage extends React.Component {
 }
 ```
 
-> Note:
+> Nota:
 >
-> Web Components often expose an imperative API. For instance, a `video` Web Component might expose `play()` and `pause()` functions. To access the imperative APIs of a Web Component, you will need to use a ref to interact with the DOM node directly. If you are using third-party Web Components, the best solution is to write a React component that behaves as a wrapper for your Web Component.
+> Los `Web Component` a menudo exponen una API imperativa. Por ejemplo, un `Web Component` `video` podría exponer las funciones `play()` y `pause()`. Para acceder a un API imperativa de un `Web Component`, necesitarás utilizar una referencia para interactuar con el DOM directamente. Si estás utilizando `Web Component` de terceros, lo mejor sería escribir un componente React que sirva como un contenedor para tu `Web Component`.
```

Comment:
* Los `Web Components` a menudo... use plural.
* Por ejemplo, un Web Component `video` ... in this case Web Component is not enclosed by ` (backtick) in the original text.
* Si estás utilizando `Web Components` plural. :)

Reply:
You're right, thanks for the suggestions, a point, web components for the closure between (backtick), not to make a translation about it, keep the element that is spoken in the section in context.  @carburo  it is possible that this is fine as well, or I do translation on it too.